.github/workflows/ci.yml

```diff
@@ -42,62 +42,20 @@ jobs:
   spell-check:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout Parent Repo
-        uses: actions/checkout@v2
-        with:
-          ref: main
-          repository: aws/aws-iot-device-sdk-embedded-C
-          path: main
       - name: Clone This Repo
         uses: actions/checkout@v2
+      - name: Run spellings check
+        uses: FreeRTOS/CI-CD-Github-Actions/spellings@main
         with:
-          path: ./tcp
-      - name: Install spell
-        run: |
-          sudo apt-get install spell
-          sudo apt-get install util-linux
-      - name: Check spelling
-        run: |
-          PATH=$PATH:main/tools/spell
-          # Make sure that the portable directory is not included in the spellcheck.
-          sed -i 's/find $DIRNAME/find $DIRNAME -not -path '*portable*'/g' main/tools/spell/find-unknown-comment-words
-          find-unknown-comment-words --directory tcp/ --lexicon tcp/.github/lexicon.txt
-          if [ "$?" = "0" ]; then
-            exit 0
-          else
-            exit 1
-          fi
-
+          path: ./
   formatting:
-    # Use only 18.04 since we want the uncrustify version to
-    # be 0.66.1_f to ensure proper formatting.
     runs-on: ubuntu-20.04
-    container: ubuntu:18.04
     steps:
-      - uses: actions/checkout@v3
-      - name: Install Uncrustify
-        run: apt-get update && apt-get install uncrustify
-      - name: Run Uncrustify
-        run: |
-          uncrustify --version
-          find . -iname "*.[hc]" -exec uncrustify --check -c tools/uncrustify.cfg {} +
-          if [ "$?" = "0" ]; then
-            exit 0
-          else
-            echo -e "\033[31;1;43mFormatting check (using Uncrustify) failed...\033[0m"
-            echo -e "\033[32;3mTo have the code uncrustified for you, please comment '/bot run uncrustify' (without the quotes) on the Pull Request.\033[0m"
```

Comment:
I would still like for this to be printed out in case of failure. This repo has a workflow which uncurstifies the code when you comment that.

Reply:
That bot would need to be updated to use the new version of uncrustify. I also dislike that this is the only repo out of all of ours that offers that as an option. I'm going to talk to @A-Zaba about getting this added into the repo and maybe tying it into this one.